frontend/src/templates/Header/PublicHeader.tsx

```diff
@@ -0,0 +1,79 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+import { chakra, Flex, FlexProps, HStack } from '@chakra-ui/react'
+
+import { ReactComponent as BrandHortSvg } from '~assets/svgs/brand/brand-hort-colour.svg'
+import { ReactComponent as BrandMarkSvg } from '~assets/svgs/brand/brand-mark-colour.svg'
+import Button from '~components/Button'
+import Link from '~components/Link'
+
+const BrandHortLogo = chakra(BrandHortSvg)
+const BrandSmallLogo = chakra(BrandMarkSvg)
+
+type PublicHeaderLink = {
+  label: string
+  href: string
+  showOnMobile?: boolean
+}
+
+export interface PublicHeaderProps {
+  /** Footer links to display, if provided. */
+  publicHeaderLinks?: PublicHeaderLink[]
+}
+
+export const PublicHeader = ({
+  publicHeaderLinks,
+}: PublicHeaderProps): JSX.Element => {
+  return (
+    <PublicHeader.Container>
+      <Link title="Form Logo" href="https://form.gov.sg/">
+        <BrandSmallLogo display={{ base: 'block', sm: 'none' }} w="40px" />
+        <BrandHortLogo display={{ base: 'none', sm: 'block' }} w="124.25px" />
```

Comment:
Should we utilize ChakraUI's [`useBreakpointValue`](https://chakra-ui.com/docs/styled-system/utility-hooks/use-breakpoint-value) hook to dynamically load the logo?

```tsx
const logoToRender = useBreakpointValue({
  base: <BrandSmallLogo w="40px" />,
  sm: <BrandHortLogo w="124px" />
})

/// Usage
<PublicHeader.Container>
   ...
   {logoToRender}
</PublicHeader.Container>
```